sigs.yaml

```diff
@@ -1743,8 +1743,8 @@ sigs:
   label: scheduling
   leadership:
     chairs:
-    - github: bsalamat
-      name: Bobby (Babak) Salamat
+    - github: ahg-g
```

Comment:
You can add @bsalamat as an `emeritus_lead` :)

Example:

https://github.com/kubernetes/community/blob/ee6617c5b7d495b1bf9998c52fabe08143cee3c5/sigs.yaml#L564-L567

Reply:
done :)